### PR TITLE
feat: enhancement: customerio option to send page name or not

### PR DESCRIPTION
**Pull request by anantjain45823**
## PR Description

Providing an option for user from config dashbaord to send page name or not in page event.
If not send customerIo automatically captures the page url and map it to the page name.

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

**Review by ItsSudip — APPROVED** [review]

File: src/integrations/CustomerIO/browser.js [hunk 1]
```diff
@@ -11,6 +13,7 @@ class CustomerIO {
     this.analytics = analytics;
     this.siteID = config.siteID;
     this.apiKey = config.apiKey;
+    this.sendPageNameInSDK = config.sendPageNameInSDK;
```
Comment (mihir-4116) [inline src/integrations/CustomerIO/browser.js:16]: add default value

File: src/integrations/CustomerIO/browser.js [hunk 2]
```diff
@@ -69,9 +76,12 @@ class CustomerIO {
 
   page(rudderElement) {
     logger.debug('in Customer IO page');
-
-    const name = rudderElement.message.name || rudderElement.message.properties.url;
-    window._cio.page(name, rudderElement.message.properties);
+    if (this.sendPageNameInSDK === false) {
+      window._cio.page(rudderElement.message.properties);
```
Comment (mihir-4116) [inline src/integrations/CustomerIO/browser.js:80]: have you validated this in customerio dashboard?

**Review by mihir-4116 — CHANGES_REQUESTED** [review]

File: src/integrations/CustomerIO/browser.js (hunk 1 above)
Comment (anantjain45823) [reply src/integrations/CustomerIO/browser.js:16]: default value is being sent from config itself

File: src/integrations/CustomerIO/browser.js (hunk 2 above)
Comment (anantjain45823) [reply src/integrations/CustomerIO/browser.js:80]: Yes

**Review by mihir-4116 — APPROVED** [review]

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**